src/singularity.ts

```diff
@@ -1734,11 +1769,27 @@ const getAvailablePerksDescription = (singularityCount: number): string => {
 function formatPerkDescription(perkData: ISingularityPerkDisplayInfo, singularityCount: number): string {
   const singTolerance = getFastForwardTotalMultiplier()
   const isNew = (singularityCount - perkData.lastUpgraded <= singTolerance)
-  const levelInfo = perkData.currentLevel > 1 ? `<br>${i18next.t('general.level')} ${perkData.currentLevel}` : ''
   //const acquiredUpgraded = ' / Acq ' + perkData.acquired + ' / Upg ' + perkData.lastUpgraded;
-  return `<span${isNew?' class="newPerk"':' class="oldPerk"'} title="${perkData.description}">${perkData.name}${levelInfo}</span>`
+  return `<span${isNew?' class="newPerk"':' class="oldPerk"'} id="${perkData.ID}"> 
+  <img src="Pictures/Default/perk${perkData.ID}.png" loading="lazy">
+  ${perkData.name}
+  </span>`
 }
 
+// Attach EventListener to perks to allow for on hover capabilities.
+export function addPerkEventHandlers() {
+  for (const perk of singularityPerks) {
+    document.getElementById(perk.ID)!.addEventListener('mouseover', () => {
```

Comment:
is there a reason to use document.getElementById?

Reply:
This is required due to caching, with a standard DOMCacheGetOrSet implementation the description box breaks whenever the tab has to reload due to switching to another primary tab. 
The discussion in the progamming channel on the discord server goes into a bit more detail about it. https://discord.com/channels/677271830838640680/705896428287033375/1111390816208953437